bears/perl/PerlCriticBear.py

```diff
@@ -1,36 +1,33 @@
-import re
-
-from coalib.bearlib.abstractions.Lint import Lint, escape_path_argument
-from coalib.bears.LocalBear import LocalBear
+from coalib.bearlib.abstractions.Linter import linter
 from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+import shutil
```

Comment:
I am not sure about the order. Might wanna refer core modules. :P
On 25 May 2016 03:22, "Zatreanu Adrian-Gabriel" <notifications@github.com>
wrote:

> In bears/perl/PerlCriticBear.py
> <https://github.com/coala-analyzer/coala-bears/pull/457#discussion_r64480001>
> :
>
> >  from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
> > +import shutil
>
> actually i think I should place this first, right? instead of import re
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/coala-analyzer/coala-bears/pull/457/files/2ddc79514cf3e6439b70cc8809ddf108135bf049#r64480001>
>